Sources/ApodiniVaporSupport/ApodiniError+AbortError.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ApodiniError+AbortError.swift
+//  
+//
+//  Created by Max Obermeier on 30.06.21.
+//
+
+import Apodini
+import Vapor
+
+// MARK: AbortError
+
+extension ApodiniError: AbortError {
```

Comment:
I assume this solves https://github.com/Apodini/Apodini/issues/217 right? just to keep track of it.

Reply:
Not entirely. REST exporter still has to make sure it converts all errors thrown to an `ApodiniError` using `Error.apodiniError`, but that should be not a big problem. I can include that with this PR.

Reply:
Not really sure on what exactly we decided on in the meeting today? I think Apodini should not make the decision for the exporter to automatically map all errors to ApodiniError.
But would it make sense for REST to map all errors to ApodiniErrors? Not sure if this would result in strangely differing output. Would there be even a difference?

I have no strong opinion on this anyways.